### PR TITLE
[C-1261] Android sign on overflow

### PR DESCRIPTION
**Pull request by sliptype**
### Description

* Improve layout of sign on screen so there is no overflow on android devices
* Update `windowSoftInputMode` so that the android keyboard overlays on top of the app, similar to ios

Before
before

After
after

> Side note: does the space between the email input and the sign up button seem too large to anyone else?

### Dragons

THIS TOUCHES SIGN ON

Also changing the keyboard mode could be dangerous, but I thoroughly tested cases where it could be problematic (outlined below)

### How Has This Been Tested?

Tested on pixel 3 which was exhibiting the problem, as well as OnePlus 7T and ios devices

Tested scroll behavior on android with new keyboard mode on
* Search
* Edit Profile
* Create / edit playlist

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
         android:screenOrientation="portrait"
-        android:windowSoftInputMode="adjustResize">
```
Comment (dylanjeffers) [inline packages/mobile/android/app/src/main/AndroidManifest.xml:38]: coolio, so this makes it similar to ios? we double checked search, filters, and edit-profile?

File: packages/mobile/src/screens/signon/SignOn.tsx [hunk 2]
```diff
@@ -732,46 +730,48 @@ const SignOn = ({ navigation }: SignOnProps) => {
...
-                  setShowInvalidEmailError(!isValidEmailString(email))
-                  // wait a bit for email validation to come back
-                  setTimeout(() => setAttemptedEmail(true), 1000)
-                }
-              }}
-            />
-            {passwordInputField()}
-            {errorView()}
-            <MainButton isWorking={isWorking} isSignin={isSignin} />
+            <SafeAreaView edges={['top']} style={styles.drawerContent}>
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/signon/SignOn.tsx:733]: nice work figuring out what works here

**Review by dylanjeffers — APPROVED** [review]
love these improvements, was bugging me for a while

File: packages/mobile/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (sliptype) [reply packages/mobile/android/app/src/main/AndroidManifest.xml:38]: Yes exactly. I check those, what do you mean by filters?

**Review by nicoback2 — APPROVED** [review]

**Pull request merged**